python/setup.py

```diff
@@ -214,14 +214,21 @@ def get_triton_cache_path():
 def get_thirdparty_packages(packages: list):
     triton_cache_path = get_triton_cache_path()
     thirdparty_cmake_args = []
+    offline_build = os.environ.get("TRITON_OFFLINE_BUILD", False)
```

Comment:
Could you add some comments documenting the meaning of this flag as the name itself didn't make it obvious to me at first.

Reply:
Moved the bool into a 0-slot function with a docstring